CHANGELOG.md

```diff
@@ -10,11 +12,13 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### 📈 Features/Enhancements
 
+- [MD] Support legacy client for data source ([#2204](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2204))
+
 ### 🐛 Bug Fixes
 
 ### 🚞 Infrastructure
 
-* Add CHANGELOG.md and related workflows ([#2414](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2414))
+- Add CHANGELOG.md and related workflows ([#2414](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2414))
```

Comment:
Why are we changing the format of the changelog? This is going to cause merge conflicts with all other PR's that are in flight and pending reviews

Reply:
prettier plugin in VScode auto formatted this. I can revert it by disabling prettier on that file. Also wonder isn't the md format defined in our `.prettierrc`

Reply:
fixed